crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -414,17 +367,14 @@ pub mod tests {
     #[test]
     fn test_qml_metadata_named() {
         let item: ForeignTypeIdentAlias = parse_quote! {
-            #[cxx_qt::qobject(qml_uri = "foo.bar", qml_version = "1", qml_name = "MyQmlElement")]
-            type MyNamedObject = super::MyObjectRust;
+            #[cxx_qt::qobject(qml_element = "OtherName")]
```

Comment:
ah i see you've put it within the `qml_element = "T"`

Reply:
Slight bit of magic but probably makes sense and avoids doing `qml_element, qml_named_element = "OtherName"` which is repetitive.